fms/models/gpt_bigcode.py

```diff
@@ -30,6 +30,8 @@ class GPTBigCodeConfig(ModelConfig):
     emb_dropout: float = 0.0
     multiquery_attn: bool = True
     ln_eps: float = 1e-5
+    linear_config: Optional[Mapping[str, Any]] = None  # pass as {"linear_type": str, <other kwargs>}
+    unfuse_strategy: Optional[str] = None  # TODO: could be an Enum
```

Comment:
this probably shouldn't be part of the config of the model, I would just pass it as a param of `__init__` directly, as it's not something directly tied to the serialization of the model but more to how we are running it

Reply:
I haven't changed this yet. We need to clarify if it is worth adding `unfuse_strategy` as argument to the model instead of it being part of `self.config` (as currently done). As a separate argument, it would be one additional argument to propagate to MHA, FFN, GLU. Passing it via `config` seems cleaner.